Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.Options.cs

```diff
@@ -56,7 +57,18 @@ public DataTemplate OptionsItemTemplate
         /// <summary>
         /// Gets the collection used to generate the content of the option list.
         /// </summary>
-        public ItemCollection OptionsItems => _optionsListView?.Items;
+        public ItemCollection OptionsItems
```

Comment:
I guess it would be cool if we mentioned throwing the exception in the documentation.
etc 
/// <exception cref="System.Exception">Thrown when _optionsListView is null</exception>